src/System.Linq.Expressions/tests/ExceptionHandling/ExceptionHandlingExpressions.cs

```diff
@@ -137,6 +137,108 @@ public void CanCatchAndThrowNonExceptionsInterpreted()
         }
 
         [Fact]
+        public void ThrowNullThrowsNRE()
+        {
+            Action throwNull = Expression.Lambda<Action>(
+                Expression.Throw(Expression.Constant(null, typeof(Expression)))
+                ).Compile(false);
+            Assert.Throws<NullReferenceException>(throwNull);
+        }
+
+        [Fact]
+        public void ThrowNullThrowsNREInterpreted()
+        {
+            Action throwNull = Expression.Lambda<Action>(
+                Expression.Throw(Expression.Constant(null, typeof(Expression)))
+                ).Compile(true);
+            Assert.Throws<NullReferenceException>(throwNull);
+        }
```

Comment:
Rather than duplicating all of these tests, one for compiled and one for interpreted, couldn't they each just be [Theory] with [InlineData(false)] and [InlineData(true)] for a ```bool compiled``` parameter?